cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/services/ScriptsService.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.lang.runtime.services;
+
+import io.cloudslang.lang.entities.bindings.ScriptFunction;
+import org.apache.commons.io.IOUtils;
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class ScriptsService {
+
+    private static final Logger logger = Logger.getLogger(ScriptsService.class);
+
+    private Map<ScriptFunction, String> scriptsMap = new HashMap<>();
+
+    @PostConstruct
+    public void init() {
+        List<ScriptFunction> list = Arrays.asList(
+                ScriptFunction.GET,
```

Comment:
This is again related to my first comment. We have `GET` here, but it's also matched individually

Reply:
All the CS functions implementations are now in files. This service loads the python code of the functions from files.